Intersect.Server/Migrations/Game/20220628180051_AddedCameraShakeInterval.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Intersect.Server.Migrations.Game
+{
+    public partial class AddedCameraShakeInterval : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "CameraShakeInterval",
+                table: "Animations",
+                nullable: false,
+                defaultValue: 0);
```

Comment:
We still have to get rid of a couple of 'defaultValue' column properties in order to allow migrations to MySQL, adding more to it is should be a no no since we want to get rid of this issue eventually.

Reply:
What issue are you referring to here? @Arufonsu

Reply:
After discussion with Aru in Discord, I think that this defaultValue is ok because it is for an int column type, while the migration error is for BLOB, TEXT, GEOMETRY, and JSON column types.

This doesn't however change all of the other comments I made last night.

Reply:
Yup, like @lodicolo , this was discussed in discord, my comment doesn't applies to int column types so feel free to ignore this comment review @lucas-tardivo 👍🏽